KA-Flocking/Assets/Scripts/Unit Scripts/Infantry.cs

```diff
@@ -8,13 +8,13 @@ public class Infantry : Unit
 
     public ContextFilter attackFilter;
     [Range(1f,1000f)]
-    public float health = 100f;
+    private float health = 100f;

```

Comment:
Why make this private? This is suppose to be public so that you don't have to go into the file to change values, but can instead change from the unity editor. Furthermore this breaks all functionality with scouts and with different types of troops.